src/components/ImportOnyxState/utils.ts

```diff
@@ -0,0 +1,53 @@
+import cloneDeep from 'lodash/cloneDeep';
+import type {UnknownRecord} from 'type-fest';
+import ONYXKEYS from '@src/ONYXKEYS';
+
+// List of Onyx keys from the .txt file we want to keep for the local override
+const keysToOmit = [ONYXKEYS.ACTIVE_CLIENTS, ONYXKEYS.BETAS, ONYXKEYS.FREQUENTLY_USED_EMOJIS, ONYXKEYS.NETWORK, ONYXKEYS.CREDENTIALS, ONYXKEYS.SESSION, ONYXKEYS.PREFERRED_THEME];
```

Comment:
I think it would make sense to export betas as well, since having or not having a beta may be the source of a bug, was this discussed somewhere?

Reply:
can betas differ across different accounts? If yes, let's definitely import them.

I was checking it on a few accounts of mine and each one of them had the same array of betas, that's why my initial thought was that it's safe to omit it

Reply:
Ah right, it can definitely be different sometimes